examples/server/tests/features/steps/steps.py

```diff
@@ -1045,7 +1056,17 @@ def start_server_background(context):
     if 'SERVER_LOG_FORMAT_JSON' not in os.environ:
         server_args.extend(['--log-format', "text"])
     print(f"starting server with: {context.server_path} {server_args}\n")
+    flags = 0
+    if 'nt' == os.name:
+        flags |= 0x00000008  # DETACHED_PROCESS
+        flags |= 0x00000200  # CREATE_NEW_PROCESS_GROUP
+        flags |= 0x08000000  # CREATE_NO_WINDOW
```

Comment:
Why hard-code these values instead of using the appropriate built-in constants?
```suggestion
        flags |= subprocess.DETACHED_PROCESS
        flags |= subprocess.CREATE_NEW_PROCESS_GROUP
        flags |= subprocess.CREATE_NO_WINDOW
```

Reply:
I should stop stackoverflow.com and use codellama ;) Thanks, I appreciate someone finally looking at the tests code